.travis.yml

```diff
@@ -13,56 +13,84 @@ notifications:
 cache:
   directories:
     - $HOME/.composer/cache
-    - $HOME/.phpbrew
+    - $HOME/.jest-cache
     - $HOME/.npm
-
-before_install:
-  - nvm install && nvm use
-  - npm install npm -g
+    - $HOME/.nvm/.cache
+    - $HOME/.phpbrew
+    - node_modules
 
 branches:
   only:
     - master
 
+before_install:
+  - nvm install
+
 jobs:
   include:
-    - stage: test
+    - name: JS unit tests
       env: WP_VERSION=latest
+      before_install:
+        - nvm install --latest-npm
+      install:
+        - npm install
       script:
-        - npm install || exit 1
-        - npm run ci || exit 1
```

Comment:
It's no longer used once removed from here https://github.com/WordPress/gutenberg/blob/7ea50218a1de9cd00aceb0a58b4c879719f26d9b/package.json#L155

Reply:
Looks like `npm run test-unit:ci`is also useless? Unless we replace the next line `npm run test-unit -- --ci --maxWorkers=2 --cacheDirectory="$HOME/.jest-cache"` with it?

Reply:
Yes, and stting a `--maxWorkers` value vs `--runInBand` can be troublesome on Travis CI

https://jestjs.io/docs/en/troubleshooting#tests-are-extremely-slow-on-docker-and-or-continuous-integration-ci-server

I suspect we have only 2 cores as we use Travis-CI.org rather than Travis-CI.com, some benchmark testing in a follow-up PR would be worthwhile to measure this.

Reply:
I experimented with this yesterday and `--maxWorkers=2` was slightly faster on our CI environment. Again, this is something we'll want to re-evaluate periodically as our project evolves.